sopel/modules/url.py

```diff
@@ -185,40 +226,21 @@ def check_callbacks(bot, trigger, url, run=True):
 
 
 def find_title(url, verify=True):
-    """Return the title for the given URL."""
```

Comment:
No reason to remove this; it still describes what the function is doing, right?

Reply:
Pardon my obtuseness from the other day, @sean-public. I couldn't see past all the red deleted lines below and find the new docstring. :sweat_smile:

Reply:
I added a new one in 8d8b1c3 that describes how it works now.